src/plugins/controls/public/options_list/components/options_list_popover.test.tsx

```diff
@@ -118,6 +118,43 @@ describe('Options list popover', () => {
     expect(sortButton.prop('disabled')).toBe(true);
   });
 
+  test('test single invalid selection', async () => {
+    const popover = await mountComponent({
+      explicitInput: {
+        selectedOptions: ['bark', 'woof'],
+      },
+      componentState: {
+        availableOptions: {
+          bark: { doc_count: 75 },
+        },
+        validSelections: ['bark'],
+        invalidSelections: ['woof'],
+      },
+    });
+    const validSelection = findTestSubject(popover, 'optionsList-control-selection-bark');
+    expect(validSelection.text()).toEqual('bark75');
+    const title = findTestSubject(popover, 'optionList__ignoredSelectionLabel').text();
+    expect(title).toEqual('Ignored selection');
```

Comment:
Because I refactored the `optionsListPopoverGetAvailableOptions`, it no longer returns `"Ignored selection" / "Ignored selections"` as part of the selections array. So, I wanted to ensure that I added a unit test to ensure that the singular/plural cases are still respected :+1: